src/app/views/collections/validation/collectionValidation.js

```diff
@@ -7,17 +7,27 @@
 export default class collectionValidation {
 
     static name(name) {
+        let response = {
+            isValid: true,
+            errorMsg: ""
+        }
+
+        if (name.match(/[^a-zA-Z0-9 ]/)) {
+            response = {
+                isValid: false,
+                errorMsg: "Collections must not have (" + name.replace(/[a-zA-Z0-9 ]/g, "") + ") symbols"
```

Comment:
It might be better to change this to a positive or guiding message. E.g how to fix the error, as in what can be used "A-Z, a-z, 0-9". It could be helpful to have what characters are causing the error, but i'd put those after how to fix it